handbook/ce/index.md

```diff
@@ -25,26 +35,34 @@ The Customer Engineering team partners with the [Sales team] (../sales/index.md)
     * [Campaign demo and training materials](https://docs.google.com/document/d/1xQxhdGaudydOn5nBGIG91F6Z4VR4NwBfuKFvgbmCjJo/edit?usp=drive_web&ouid=107037782400977645523)
 
 
-## Goals
-
-TODO
-
 ## Members
 
+<!-- Alphabetically, by surname. -->
+
 - Customer Engineers
-  - [Christine Lovett](../../company/team/index.md#christine-lovett-she-her)
+  - [Emily Chapman](../../company/team/index.md#emily-chapman-she-her)
   - [Jonah Dueck](../../company/team/index.md#jonah-dueck-he-him)
-  - [Josh Saunders](../../company/team/index.md#josh-saunders)
-  - [Nick McMillen](../../company/team/index.md#nick-mcmillen-he-him)
+  - [Christine Lovett](../../company/team/index.md#christine-lovett-she-her)
   - [Mike McLaughlin](../../company/team/index.md#mike-mclaughlin-he-him)
-  - [Emily Chapman](../../company/team/index.md#emily-chapman-she-her)
+  - [Nick McMillen](../../company/team/index.md#nick-mcmillen-he-him)
 - Customer Support
   - [Virginia Ulrich](../../company/team/index.md#virginia-ulrich-she-her) (Head of Customer Support)
-  - We will be hiring support engineers soon. Our Customer Engineers will continue in their support rotation for now.
-- Training
+  - We will be hiring Customer Support Engineers soon. Our Customer Engineers will continue in their support rotation for now.
 - [Quinn Slack](../../company/team/index.md#quinn-slack) (CEO) is leading Customer Engineering for now.
   - We are hiring for VP Customer Engineering and will have the role posted very soon.
 
 ## Roles
```

Comment:
This section seems unnecessary.  Maybe we could combine it with "Members" and add a link to the role in the header along with a short description of the role with the member names below.  Alternatively, you did cover the roles in the main "Customer Engineering" section so maybe we move the definition of the roles down to this section explicitly.

Reply:
I will remove the separate roles page and inline them here.